services/ui-src/src/components/drawers/ReportDrawer.tsx

```diff
@@ -26,7 +27,7 @@ export const ReportDrawer = ({
         id={form.id}
         formJson={form}
         onSubmit={onSubmit}
-        formData={report}
+        formData={formData ?? report}
```

Comment:
Thought: I don't think we'll need report to ever be passed here directly in the future, but that's a consideration for future us.

Reply:
Yeah we might be able to refactor this later. For now there are other areas that follow a similar pattern so we'll wait on that